gems/pending/postgres_ha_admin/failover_databases.rb

```diff
@@ -51,6 +51,14 @@ def query_repmgr(connection)
       result
     end
 
+    def symbolize_record_keys(record)
+      new_record = {}
+      record.each do |k, v|
+        new_record[k.to_sym] = v
+      end
+      new_record
```

Comment:
Both of the following seem to do the right thing:
`record.each_with_object({}) { |(k, v), hash| hash[k.to_sym] = v }`
and
`Hash[record.map {|k, v| [k.to_sym, v]}]`